modules/manage/pages/cluster-maintenance/disk-utilization.adoc

```diff
@@ -306,7 +306,12 @@ You can set a soft limit for a minimum free disk space alert. This soft limit ge
 
 NOTE: The alert threshold can be set in either bytes or percentage of total space. To disable one threshold in favor of the other, set it to zero.
 
-When a disk exceeds the set threshold, `redpanda_storage_disk_free_space_alert` updates, and an error message is written to the Redpanda service log.
+When a disk exceeds the configured threshold, Redpanda updates the xref:reference:public-metrics-reference.adoc#redpanda_storage_disk_free_space_alert[`redpanda_storage_disk_free_space_alert`] metric appropriately and writes an error level `storage space alert` message to the service log. The message looks like the following:
+
+[,bash]
+----
+ERROR 2023-12-08 15:07:45,716 [shard  0] cluster - storage space alert: free space at 25.574% on /var/lib/redpanda/data: 96.732GiB total, 24.739GiB free, min. free 0.000bytes. Please adjust retention policies as needed to avoid running out of space
```

Comment:
Hmmmm...would be nice if eng updated the msg as follows: 
```suggestion
ERROR 2023-12-08 15:07:45,716 [shard  0] cluster - storage space alert: free space at 25.574% on /var/lib/redpanda/data: 96.732GiB total, 24.739GiB free, min. free 0.000bytes. Be sure to adjust retention policies as needed to avoid running out of space.
```

Reply:
I wouldn't include this update if that's the message that the system outputs. If we include errors messages in docs it should be 100% the same as the product